### DISCUSSION TITLE
Custom Address Phone Field is not showing on cp edit order screen

### DISCUSSION THREAD

**Discussion by philipboomy** [craftcms/commerce#2952] [Ideas]
### Description
Editor cannot see the phone number on a order except when clicking edit billing address via the order. This a big step back from V3. Having to click 2 time to see phone number is not use friendly. In addition to that only being able to see the phone number when editing is dangerous.

### Steps to reproduce

1. Have a custom phone checkout field.2. 
2. Do you use saved address when checking out. Meaning submit address as part of checkout.
3. After checkout go to order and there you can see that phone is not visible.

Would be nice to have it appear below the billing address like in V3.

### Craft CMS version

4.2.2

### Craft Commerce version

4.1.0

### Installed plugins and versions

-

**Comment by lukeholder** [comment]
We would need to support letting developers tweaking the address formatter for the order edit page. Added to our backlog (COM-214).
